src/shared/helpers/__tests__/test-start-time.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { getNewTestStartTime, isValidStartDate } from '../test-start-time';
+
+describe('Test Start Time helper functions', () => {
+
+  describe('getNewTestStartTime', () => {
+    it('should return the correct date', () => {
+      const inputDate = '2021-01-19';
+      const startDateTime = '2020-12-25T08:10:00';
+      expect(getNewTestStartTime(inputDate, startDateTime)).toBe('2021-01-19T08:10:00');
+    });
+  });
+
+  describe('', () => {
```

Comment:
empty describe?

Reply:
Ohh, the result of TDD hah

Added the name of the function under test